### ISSUE TITLE
the error about build environment?

### ISSUE THREAD

**Issue by younkun** [avanetten/simrdwn#58]
when the dockerfile exectue the line 99(   conda install -n simrdwn2 -c defaults tensorflow-gpu=1.13.1),  it always have a error ,sloving the environment failed ,when i delete the line 88 (conda update conda). the error disappeared.

**Comment by bruceche11** [comment]
> when the dockerfile exectue the line 99( conda install -n simrdwn2 -c defaults tensorflow-gpu=1.13.1), it always have a error ,sloving the environment failed ,when i delete the line 88 (conda update conda). the error disappeared.

really?Have you succeed in building?

**Comment by younkun** [comment]
no,i have not , it also has some problems,When I execute the line 139(opencv=4.0.0)

**Comment by bruceche11** [comment]
https://hub.docker.com/r/dzynegca/simrdwn
I am pulling this. Maybe you can have a try too.

**Comment by younkun** [comment]
thank you very much!

**Comment by jacquesmats** [comment]
Hey @younkun, can you share the error? We have experienced some error in the Dockerfile and solved them on #52 and #53.

**Comment by younkun** [comment]
@jacquesmats Hello ,I have the same problems , the line 139 opencv=4.0.0 ,There is a conflict between opencv and gdal

**Comment by jacquesmats** [comment]
I see. Did you follow the instructions on #53 ? It should work. Don't use conda forge and install just opencv not specifiying the version.
________________________________

**Comment by avanetten** [comment]
The most recent dockerfile builds fine as of 19 July 2019.  Give that a shot.  The temporary fix in  [#53](https://github.com/avanetten/simrdwn/issues/53) may build successfully, but leads to errors when running inference.
